metadeploy/context_processors.py

```diff
@@ -18,5 +18,6 @@ def env(request):
         "SITE": site_serializer.data,
         "YEAR": datetime.utcnow().year,
         "SENTRY_DSN": settings.SENTRY_DSN,
+        "DEVHUB_USERNAME": bool(settings.DEVHUB_USERNAME),
```

Comment:
This might be clearer if we called it SCRATCH_ORGS_AVAILABLE

Reply:
@jgerigmeyer I think this requires coordination with the frontend, if we want to make that change.